src/common/config.cpp

```diff
@@ -116,6 +116,21 @@ void Config::initialize(ConfigParser const& cp) {
     config_["tsv-fields"] = tsvFields;
   }
 
+  // ensures factors backward compability whilst keeping the more user friendly CLI
+  if(get<std::string>("lemma-dependency").empty()) {
```

Comment:
@pedrodiascoelho Is there a way to check if the model has factors? If so, maybe this should be added to this condition here?

Reply:
The factors are never explicit set via the cli. They are triggered if you provide a vocab file with the sufix `.fsv`, so I guess the only way would be checking if any vocab file finishes with `.fsv`, although I'm not sure if this is a clean solution.

Reply:
Thanks, then let's keep this as it is.